Unix/scripts/config_keytab_update.sh

```diff
@@ -25,6 +25,25 @@ cleanTempFile()
     rm -f $tmpfile > /dev/null 2>&1
 }
 
+purgeforever()
+{
+    # call unconfigure function
+    unconfigure
+
+    # create maker file to purge the keytab cron forever
+    if [ ! -f /etc/.omi_keytabcron_purge_forever ]; then
+        printf "create maker file to purge the keytab cron forever...\n"
+        touch /etc/.omi_keytabcron_purge_forever
+    fi
+}
+
+undopurgeforever()
+{
+    # just remove the maker file
+    printf "undo the purgeforever and remove the maker file\n"
+    rm -f /etc/.omi_keytabcron_purge_forever
+}
+
```

Comment:
Is it necessary to "purge" forever?  Why not just have a skipKeytabCron that skips once?  I don't like the idea of a hidden file in /etc.  I really maybe forever since no one knows it's there, and it's not cleaned up during uninstall.  If you really need such a file, at least put it in /etc/opt/omi?

Reply:
Why I add the option is because many customers complaint that they removed the cron, it come back again once oms upgraded, I want to provide a way for them to make them satisfy and happy.

The maker file if under /etc/opt/omi, when use "omsagent.sh --purge", it will be deleted, I don't want it to be deleted.